src/openloco/input.cpp

```diff
@@ -51,9 +51,9 @@ namespace openloco::input
     }
 
     // 0x00406FEC
-    void enqueue_mouse_button(mouse_button button)
+    void enqueue_mouse_button(int32_t button)
     {
-        ((void (*)(int))0x00406FEC)((int32_t)button);
+        ((void (*)(int))0x00406FEC)(button);
```

Comment:
Are these regressions?

Reply:
No, these changes were deliberate in eafbd687c773d2166ef9905f4140cb084be45f38. From the commit description:

> The pushed numbers are remapped to different values when popped. This explains why the pushed constants don't match their events (e.g. pushing `left_released` when right is pressed)